cernopendata/modules/fixtures/data/records/cms-validation-code-ttbar.json

```diff
@@ -0,0 +1,72 @@
+[
+{
+  "abstract": {
+    "description": " <p>This code for analysing and benchmarking of the DoebleElectron, DoubleMuon and MuEG datasets is set up at Research level, i.e. it requires university-student-level programming experience. Minimal acquaintance with Linux and the ROOT analysis package (<a href=\"https://root.cern.ch/\">https://root.cern.ch/</a>) as well as a basic text editor is needed. </p>"
+  },
+  "accelerator": "CERN-LHC",
+  "authors": [
+    {
+      "name": "Zenaiev, Oleksandr"
+    }
+  ],
+  "collections": [
+    "CMS-Validation-Utilities"
+  ],
+  "date_created": "2011",
+  "date_published": "2017",
+  "distribution": {
+    "formats": [
+      "gz"
+    ],
+    "number_files": 1,
+    "size": 1288913
+  },
+  "doi": "10.7483/OPENDATA.CMS.TTGV.3IKL",
+  "experiment": "CMS",
+    "files": [
+    {
+      "checksum": "sha1 82c1d9bc3204ee8d106acab68d1aa56a1bc3e4fb", 
```

Comment:
`sha1:82c1d9bc3204ee8d106acab68d1aa56a1bc3e4fb` please note we need `:` in between